Add LIBPTHREAD_SO macro to gnu/lib-names.h and disable nscd build
---
 nscd/Makefile         |    2 +-
 scripts/lib-names.awk |    2 ++
 2 files changed, 3 insertions(+), 1 deletion(-)

--- a/scripts/lib-names.awk
+++ b/scripts/lib-names.awk
@@ -61,6 +61,8 @@ END {
     cmd = "LC_ALL=C sort"; print lines["DEFAULT"] | cmd; close(cmd);
   }
 
+  print "#define LIBPTHREAD_SO \"libpthread.so.0.3\""
+
   print "";
   print "#endif	/* gnu/lib-names.h */"
 }
--- a/nscd/Makefile.orig
+++ b/nscd/Makefile
@@ -36,7 +36,7 @@
 		dbg_log nscd_conf nscd_stat cache mem nscd_setup_thread \
 		xmalloc xstrdup aicache initgrcache gai res_hconf
 
-ifeq ($(have-thread-library),yes)
+ifeq ($(have-thread-library),yesreallyyes)
 
 ifneq (yesyes,$(have-fpie)$(build-shared))
 others += nscd
